resources/metadata/uiVeri5ExecuteTests.yaml

```diff
@@ -0,0 +1,80 @@
+metadata:
+  name: uiVeri5ExecuteTests
+  description: Executes UI5 e2e tests using uiVeri5
+  longDescription: |
+    In this step the ([UIVeri5 tests](https://github.com/SAP/ui5-uiveri5)) are executed.
+
+#    The step is using `dockerExecute` step to spin up two containers in a Docker network:
+#    * a Selenium/Chrome container (`selenium/standalone-chrome`)
+#    * a NodeJS container (`node:lts-stretch`)
+#    In the Docker network, the containers can be referenced by the values provided in `dockerName` and `sidecarName`, the default values are `uiVeri5` and `selenium`.
+#   !!! note
+#        In a Kubernetes environment, the containers both need to be referenced with `localhost`.
+spec:
+  inputs:
+    params:
+      - name: installCommand
+        type: string
+        description: The command that is executed to install the uiveri5 test tool.
+        default: npm install @ui5/uiveri5 --global --quiet
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+      - name: runCommand
+        type: string
+        description: "The command that is executed to start the tests."
+        default: /home/node/.npm-global/bin/uiveri5
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+      - name: runOptions
+        type: "[]string"
+        description: "Options to append to the runCommand, last parameter has to be path to conf.js (default if missing: ./conf.js)."
+        default: ["--seleniumAddress='http://localhost:4444/wd/hub'"]
```

Comment:
The default depends on the landscape, localhost works well in Kubernetes but for Docker execution using a network we need a name.

Reply:
How to we provide such dependent defaults?

Reply:
As we expect most of our use cases to be covered by this default, I suggest to leave it like that. 

I'll add comments to the description.